### PR TITLE
packages: add support to build with docker

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: .github/workflows/debian.yml [hunk 1]
```diff
@@ -46,10 +46,18 @@ jobs:
...
         run: |
-          cd packages
-          dch --newversion $(cat ../base_version)-1 "New upstream version"
+          OLD_RELEASE=$(grep -E -o '[0-9.]+' packages/debian/changelog | \
+                          head -n1)
+          OLD_RELEASE_DATE_FULL="$(grep '^ -- ' packages/debian/changelog | \
+                                     head -n1 | ^
+                                     sed -E -e 's/ -- .+<[^>]+>  //')"
+          OLD_RELEASE_DATE=$(date --date="${OLD_RELEASE_DATE}" +%Y-%m-%d)
+          make update-latest-release \
```
Comment (kenhys) [inline .github/workflows/debian.yml]:
```
OLD_RELEASE_DATE=$(date --date="${OLD_RELEASE_DATE_FULL}" +%Y-%m-%d)
```

Should it be OLD_RELEASE_DATE_FULL?

Comment (kou) [reply .github/workflows/debian.yml]: Good catch.

File: .github/workflows/debian.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          git clone --depth 1 https://github.com/apache/arrow.git
+          git clone --depth 1 https://github.com/groonga/groonga.org.git
+          git clone --depth 1 https://github.com/clear-code/cutter.git
+      - name: Generate configure
+        run: |
+          ./autogen.sh
+      - name: Configure for archive
+        run: |
+          ./configure \
+            --cutter-source-path=../cutter \
```
Comment (kenhys) [inline .github/workflows/debian.yml]: It seems that --with-cutter-source-path is valid

Comment (kenhys) [reply .github/workflows/debian.yml]: I've fixed this commit.

**Pull request merged**